meerk40t/gui/wxmeerk40t.py

```diff
@@ -650,9 +651,10 @@ def handleGUIException(exc_type, exc_value, exc_traceback):
     except:
         pass
 
-    error_log = "MeerK40t crash log. Version: %s on %s - %s\n" % (
+    error_log = "MeerK40t crash log. Version: %s on %s:%s - %s\n" % (
         APPLICATION_VERSION,
-        sys.platform,
+        platform.system(),
+        platform.machine(),
```

Comment:
first functional change... improvement

Reply:
Sorry - I thought I added that as a commit comment, but apparently not.